app/core.py

```diff
@@ -1,33 +1,34 @@
 """Main functionalities."""
 
 from .image.image import (Image, RawDataContainer)
-from .image.color_format import (RGB3_FORMAT, BGR3_FORMAT)
+from .image.color_format import AVAILABLE_FORMATS
 from .parser.factory import ParserFactory
 
 
-def run_core_functionality(args):
+def load_image(file_path, color_format, resolution):
 
-    image = create_Image(args)
-    parser = ParserFactory.create_object(
-        determine_color_format(args["color_format"]))
+    image = Image.from_file(file_path)
+    parser = ParserFactory.create_object(determine_color_format(color_format))
 
     image = parser.parse(image.data_buffer,
-                         determine_color_format(args["color_format"]),
-                         args["resolution"][0], args["resolution"][1])
-    displayable_data = parser.get_displayable(image)
+                         determine_color_format(color_format), resolution[0],
+                         resolution[1])
 
+    return image
 
-def create_Image(args):
 
-    raw_data = RawDataContainer.from_file(args["FILE_PATH"])
-    return Image(raw_data.data_buffer)
+def get_displayable(image):
+
+    if image.color_format is None:
+        raise Exception()
+    parser = ParserFactory.create_object(image.color_format)
+
+    return parser.get_displayable(image)
 
 
 def determine_color_format(format_string):
 
-    if format_string == "RGB3":
-        return RGB3_FORMAT
-    elif format_string == "BGR3":
-        return BGR3_FORMAT
+    if format_string in AVAILABLE_FORMATS.keys():
+        return AVAILABLE_FORMATS[format_string]
     else:
-        raise NotImplementedError
+        raise NotImplementedError()
```

Comment:
Same here, there should be any message that can be displayed i. e. "Provided string is not a name for any format from [list of formats' names].